CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ Mapbox welcomes participation and contributions from everyone.
 
 ## Mapbox Android Telemetry
 
+## v4.7.4
+This release is based off off 4.7.0 with cherry-piked future commits[#457, #455, # 444, #443] from master.
```

Comment:
`cherry-picked` but `This release is based off off 4.7.0.` should suffice IMO since the commits are listed below.